tests/legacy-cli/e2e/tests/build/rebuild-replacements.ts

```diff
@@ -0,0 +1,31 @@
+import { getGlobalVariable } from '../../utils/env';
```

Comment:
WDYT of replacing this with a new spec large in `replacements_spec_large`?

This is the one I had on my branch:

```
it('file replacements work with watch mode', (done) => {
    const overrides = {
      fileReplacements: [
        {
          replace: normalize('/src/meaning.ts'),
          with: normalize('/src/meaning-too.ts'),
        },
      ],
      watch: true,
    };

    let buildNumber = 0;

    runTargetSpec(host, browserTargetSpec, overrides, 45000, createConsoleLogger()).pipe(
      debounceTime(1000),
      tap((buildEvent) => expect(buildEvent.success).toBe(true)),
      tap(() => {
        const fileName = join(outputPath, 'main.js');
        buildNumber += 1;

        switch (buildNumber) {
          case 1:
            expect(virtualFs.fileBufferToString(host.scopedSync().read(fileName)))
              .toMatch(/meaning\s*=\s*42/);
            expect(virtualFs.fileBufferToString(host.scopedSync().read(fileName)))
              .not.toMatch(/meaning\s*=\s*10/);
            host.writeMultipleFiles({
              'src/meaning-too.ts': 'export var meaning = 84;',
            });
            break;

          case 2:
            expect(virtualFs.fileBufferToString(host.scopedSync().read(fileName)))
              .toMatch(/meaning\s*=\s*84/);
            expect(virtualFs.fileBufferToString(host.scopedSync().read(fileName)))
              .not.toMatch(/meaning\s*=\s*42/);
            break;
        }
      }),
      take(2),
    ).toPromise().then(done, done.fail);
  });
```

It should be faster than the e2e and also test the content is correct.

Reply:
Added.